From a4a4d7f9183bae11d81616346038e9efaba2fce1 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Mon, 16 Dec 2024 19:15:52 +0000
Subject: [PATCH] dtoverlays: Use continuous clock mode for ov9281

This increases the maximum frame rate from 247 to 260fps in
10-bit mode.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/ov9281-overlay.dts | 1 -
 arch/arm/boot/dts/overlays/ov9281.dtsi        | 1 -
 2 files changed, 2 deletions(-)

--- a/arch/arm/boot/dts/overlays/ov9281-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ov9281-overlay.dts
@@ -29,7 +29,6 @@
 				csi_ep: endpoint {
 					remote-endpoint = <&cam_endpoint>;
 					data-lanes = <1 2>;
-					clock-noncontinuous;
 				};
 			};
 		};
--- a/arch/arm/boot/dts/overlays/ov9281.dtsi
+++ b/arch/arm/boot/dts/overlays/ov9281.dtsi
@@ -19,7 +19,6 @@ cam_node: ov9281@60 {
 		cam_endpoint: endpoint {
 			clock-lanes = <0>;
 			data-lanes = <1 2>;
-			clock-noncontinuous;
 			link-frequencies =
 				/bits/ 64 <400000000>;
 		};
